src/main/java/duke/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+/**
+ * General category
+ */
+public class Task {
+    private String name;
+    private Boolean completed = false;
```

Comment:
Boolean variables/methods should be named to sound like booleans.
```suggestion
    private Boolean isCompleted = false;
```